Remotion/ObjectBinding/Web.Development.WebTesting.TestSite/ObjectBinding.Web.Development.WebTesting.TestSite.csproj

```diff
@@ -166,6 +134,8 @@
       <Link>Remotion LGPL.licenseheader</Link>
     </Content>
     <Content Include="packages.config" />
+    <None Include="Images\Remotion.ObjectBinding.Sample.Job.gif" />
+    <None Include="Images\Remotion.ObjectBinding.Sample.Person.gif" />
```

Comment:
those images should be moved to shared project, folder res/Image and the paths updated where the image is used.